File: Data Structures & Algorithms/anagram-groups/submission-0.py
class Solution:
    def groupAnagrams(self, strs: List[str]) -> List[List[str]]:
        count = []
        flag = []
        for i in range(len(strs)):
            count.append(Counter(strs[i]))
            flag.append(True)
        result = []
        for i in range(len(strs)):
            if flag[i]:
                temp = [strs[i],]
                flag[i] = False
                for j in range(i+1,len(strs)):
                    if count[i] == count[j]:
                        temp.append(strs[j])
                        flag[j] = False
                result.append(temp)
        return result

File: Data Structures & Algorithms/plus-one/submission-0.cpp
class Solution {
public:
    vector<int> plusOne(vector<int>& digits) {
        bool flag = true;
        int i = digits.size()-1;
        while(flag){
            if(i == -1){
                digits.insert(digits.begin(),1);
                break;
            }
            digits[i] += 1;
            if(digits[i] == 10){
                digits[i] = 0;
            }
            else{
                break;
            }
            i--; 
        }
        return digits;
    }
};
